script.js

```diff
@@ -17,45 +17,87 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
 /** ************** Refactoring projects section *************** */
 const projects = [
   {
-    id: 1,
+    id: '1',
     title: 'Multi-Post Stories Gain+Glory',
     languages: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
     buttonName: 'See Project',
+
+    image: './images/desktop/Snapshoot-Portfolio.svg',
+    titlePopup: 'Keeping track of hundreds of components',
+    languagesPopup: ['HTML', 'CSS', 'Javascript'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    liveLink: 'https://www.example.com/',
+    sourceLink: 'https://github.com/RuriEnciso/Portfolio-Project',
   },
 
   {
-    id: 2,
+    id: '2',
     title: 'Multi-Post Stories Gain+Glory',
     languages: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
     buttonName: 'See Project',
+
+    image: './images/desktop/Snapshoot-Portfolio.svg',
+    titlePopup: 'Keeping track of hundreds of components',
+    languagesPopup: ['HTML', 'CSS', 'Javascript'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    liveLink: 'https://www.example.com/',
+    sourceLink: 'https://github.com/RuriEnciso/Portfolio-Project',
   },
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! 👏 👏 
- [ ] Kindly note ❗ this applies to all your pops mobile pop cards included
- [ ] Please add a blur background on all the cards just as the template
- [ ] please see screenshots below.

`original template`
![image](https://github.com/RuriEnciso/Portfolio-Project/assets/66230306/6b6d2c5c-d2ec-4924-ac8b-0d726f8d796d)

`your template`
![image](https://github.com/RuriEnciso/Portfolio-Project/assets/66230306/9718e21e-0f65-43c3-afc3-f857615b7efe)